aladdin/bash/container/create-namespace/create-namespace

```diff
@@ -3,7 +3,9 @@ set -eu -o pipefail
 
 function create_namespace {
     kubectl create namespace $1
-    kubectl label --overwrite namespace $1 owner=$2 || true
+    if [ -z "$2" ]; then
+    	kubectl label --overwrite namespace $1 owner=$2 || true
```

Comment:
only set the label if we have an owner (arg 2)

Reply:
good catch